### ISSUE TITLE
When using albumentations, there is a deterioration of the mAP.

### ISSUE THREAD

**Issue by DP1701** [ultralytics/yolov5#7815]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

Hello everyone, maybe someone can help me understand this case. 

I have trained two different models. One with the additional augmentation function (albumentations) and one without. 

I would assume that the additional augmentation with the albumentation function leads to better results (better precision, recall and mAP). But now I have the case that the recall and the mAP turn out worse when I use albumentations. However, the loss curve for the validation shows that no overfitting occurs when using albumentations. Also the loss values for the val curves are slightly better when I use albumentations. 

I would like to know why the mAP deteriorates when I train with albumentations augmentation.

Train:
```
python train.py --img 1280 --batch -1 --epochs 300 --data own_data.yaml --weights yolov5m6.pt
```

Dataset: 
Training: 10,187 images (3 object classes).
Validation: 1748 images
Test: 1054 images

Red (no use of albumentations)
Grey (Albumentations was used)
<img width="1050" alt="1" src="https://user-images.githubusercontent.com/68444979/168449569-cceaa3b5-3ff8-4ae5-bad1-ae24ad5bffa3.png">
<img width="1035" alt="2" src="https://user-images.githubusercontent.com/68444979/168449538-f86e0cb3-9069-49ef-9862-bd663ceb61b4.png">

**Comment by glenn-jocher** [comment]
@DP1701 all part of the game. Clearly too little or too much augmentation is poor, the challenge is finding the right augmentations in the right magnitude to improve the robustness in real world deployments.
